### PR TITLE
[KOGITO-1874] Send notif if images nightly success

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-1874

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by radtriste** [comment]
tested ok for nightly and release process

File: Jenkinsfile.deploy [hunk 1]
```diff
@@ -37,18 +38,22 @@ pipeline {
...
+        booleanParam(name: 'CREATE_CHANGES_PR', defaultValue: false, description: 'In case of not releasing, you can ask to create a PR with the changes')
 
         // Bot author information. Set as params for easy testing.
         string(name: 'GIT_AUTHOR_BOT', defaultValue: 'bsig-gh-bot', description: 'From which author should the PR be created ?')
         string(name: 'BOT_CREDENTIALS_ID', defaultValue: 'bsig-gh-bot', description: 'Credentials for PR creation')
     }
 
     environment {
+        KOGITO_CI_EMAIL_TO = credentials('KOGITO_CI_EMAIL_TO')
+
```
Comment (Kaitou786) [inline Jenkinsfile.deploy:56]: Is the purpose of this extra space to group similar var?

File: Jenkinsfile.deploy [hunk 2]
```diff
@@ -141,6 +145,12 @@ pipeline {
                     sh 'cat modules/kogito-management-console/module.yaml'
                     sh 'cat modules/kogito-task-console/module.yaml'
                     sh 'cat modules/kogito-trusty-ui/module.yaml'
+
+                    if(isThereAnyChanges()){
```
Comment (Kaitou786) [inline Jenkinsfile.deploy]:
```suggestion
                    if (isThereAnyChanges()) {
```

File: Jenkinsfile.deploy [hunk 3]
```diff
@@ -282,16 +281,26 @@ pipeline {
...
+                    def commitMsg = "[${getBuildBranch()}] Update Maven artifacts"
+                    def prBody = "Generated by build ${BUILD_TAG}: ${BUILD_URL}."
+                    if(isRelease()) {
+                        commitMsg = "[${getBuildBranch()}] Update project version to ${getProjectVersion()} for release"
+                        prBody += "\nPlease do not merge, it will be merged automatically after testing."
+                    }
+                    String prLink = githubscm.createPR(commitMsg, prBody, getBuildBranch(), getBotAuthorCredsID())
+                    deployProperties['kogito-images.pr.link'] = prLink
+
+                    if(isCreateChangesPR()) {
```
Comment (Kaitou786) [inline Jenkinsfile.deploy]:
```suggestion
                    if (isCreateChangesPR()) {
```

**Review by Kaitou786 — APPROVED** [review]
Small thing, otherwise looks good to me :)

File: Jenkinsfile.deploy (hunk 1 above)
Comment (radtriste) [reply Jenkinsfile.deploy:56]: yep. find it more readable

**Review by spolti — APPROVED** [review]

**Pull request merged**